[2025-02-19T20:12:10.162+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-19T20:12:10.196+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag_example_v0.first_task scheduled__2025-01-28T00:00:00+00:00 [queued]>
[2025-02-19T20:12:10.220+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag_example_v0.first_task scheduled__2025-01-28T00:00:00+00:00 [queued]>
[2025-02-19T20:12:10.221+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-19T20:12:10.376+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-01-28 00:00:00+00:00
[2025-02-19T20:12:10.407+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1273) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-19T20:12:10.415+0000] {standard_task_runner.py:72} INFO - Started process 1295 to run task
[2025-02-19T20:12:10.414+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag_example_v0', 'first_task', 'scheduled__2025-01-28T00:00:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/wtv.py', '--cfg-path', '/tmp/tmpdvnw3w27']
[2025-02-19T20:12:10.425+0000] {standard_task_runner.py:105} INFO - Job 36: Subtask first_task
[2025-02-19T20:12:10.561+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag_example_v0.first_task scheduled__2025-01-28T00:00:00+00:00 [running]> on host 8fb7a5664bcb
[2025-02-19T20:12:10.927+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='goncalo' AIRFLOW_CTX_DAG_ID='first_dag_example_v0' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-28T00:00:00+00:00'
[2025-02-19T20:12:10.939+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-19T20:12:11.000+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-19T20:12:11.037+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hello world!"']
[2025-02-19T20:12:11.094+0000] {subprocess.py:99} INFO - Output:
[2025-02-19T20:12:11.113+0000] {subprocess.py:106} INFO - Hello world!
[2025-02-19T20:12:11.119+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-19T20:12:11.223+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-19T20:12:11.224+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=first_dag_example_v0, task_id=first_task, run_id=scheduled__2025-01-28T00:00:00+00:00, execution_date=20250128T000000, start_date=20250219T201210, end_date=20250219T201211
[2025-02-19T20:12:11.424+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-19T20:12:11.488+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-19T20:12:11.492+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
